CoreFoundation/Base.subproj/CFKnownLocations.c

```diff
@@ -91,6 +91,21 @@ CFURLRef _Nullable _CFKnownLocationCreatePreferencesURLForUser(CFKnownLocationUs
             break;
     }
 
+#elif TARGET_OS_ANDROID
+
+    switch (user) {
+    case _kCFKnownLocationUserAny:
+    case _kCFKnownLocationUserByName:
+      abort();
+    case _kCFKnownLocationUserCurrent: {
+      const char *buffer = getenv("CFFIXED_USER_HOME");
+      CFURLRef userdir = CFURLCreateFromFileSystemRepresentation(kCFAllocatorSystemDefault, (const unsigned char *)buffer, strlen(buffer), true);
```

Comment:
I think we should ensure the environment variable is none,pry and trap?

Reply:
I was going for the environment variable is unset, the `CFURLCreateFromFSR` will abort

Reply:
Made it more explicit